index.html

```diff
@@ -4817,6 +4789,69 @@ <h2>Content Integrity Protection</h2>
 hash for the context as registered in the DID specification registries
 (see the registration process for more detail) [[?DID-SPEC-REGISTRIES]].
       </p>
+    <section>
+      <h2>Persistence</h2>
+      <p>
+    </section>
+DIDs are designed to be persistent in the sense that no administrator can
+take control away from the controller, nor can an administrator prevent 
+their use for any particular purpose such as authentication, 
+authorization, and attestation. There is no trusted third party with the
+capability of removing an individual's—or an organization's identifier—nor
+can they render it inoperable.
```

Comment:
I'm in general agreement with this paragraph, but since controllers can delegate control, I think this should be slightly tweaked. I think what's important here is that DIDs are designed to enable independent control by controllers, i.e., you get full control over your identifiers (including the ability to delegate control as you see fit), you don't just rent them from a service that has essentially delegated control to *you*, that they can later revoke as they see fit.

```suggestion
DIDs are designed to be persistent such that a controller need not rely
upon a single trusted third party or administrator to maintain their
identifiers. No administrator can take control away from the controller,
nor can an administrator prevent their identifiers' use for any particular
purpose such as authentication, authorization, and attestation. No
third party can act on behalf of a controller to remove or render
inoperable an individual's (or an organization's) identifier without the
controller's consent.
```